settings.gradle

```diff
@@ -1,4 +1,4 @@
-include ':web-did'
+include ':web-did', ':credential-status', ':ethr-status-registry'
```

Comment:
I think a slightly better name for the module would be `ethr-status` since the actual registry is the smart contract.
I think this would make the package name shorter `me.uport.sdk.ethrstatus`

I'll be switching the typescript name too